sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/TestService.java

```diff
@@ -359,7 +360,8 @@ else if (test.isBlocker() && !workItem.isBlocker())
 		}
 		else if (workItem.getId() != null && existingBug != null)
 		{
-			existingBug.setHashCode(-1);
+			int randomHashCode = RandomUtils.nextInt();
```

Comment:
Please implement separate method that will trigger RandomUtils until int is not equals to existing value, I would name it nextInt(List<Integer> excluding)